kotlinx-coroutines-core/jvm/test/guide/example-compose-05.kt

```diff
@@ -9,18 +9,18 @@ import kotlinx.coroutines.*
 import kotlin.system.*
 
 fun main() = runBlocking<Unit> {
-//sampleStart
+    //sampleStart
```

Comment:
This should not have been changed (sampleEnd below, too)

Reply:
This is knit leftovers

Reply:
That's actually an indentation bug in develop branch due to 3rd party merged PR. I'll fix it in develop. Those files should not be committed as a part of _this_ PR.